app/models/event.rb

```diff
@@ -177,6 +177,81 @@ def email_addresses_of_type_without_notification_sent(type)
     applications.collect { |a| a.user.email }
   end
 
+  # Returns a string of all email addresses of accepted applications
+  #
+  # @param none
+  # @return [String] Concatenation of all email addresses of accepted applications, seperated by ','
+  def email_addresses_of_accepted_applicants
+    applications = application_letters.where(status: ApplicationLetter.statuses[:accepted])
+    applications.collect { |a| a.user.email }.join(',')
+  end
+
+  # Returns a new email to a set of participants
+  #
+  # @param criteria String
+  # @param value String
+  # @param sender String
+  # @return [Email] new email
+  def generate_participants_email(all, groups, users, sender)
+    email = Email.new
+    email.hide_recipients = false
+    email.recipients = email_addresses_of_participants(all, groups, users)
+    email.reply_to = sender
+    email.subject = ''
+    email.content = ''
+    email
```

Comment:
This could be made less verbose by passing the variables as an argument to `new`, like so
```ruby
Email.new(
  hide_recipients: false,
  stuff: more_stuff)
```